common/src/DefaultWebProxySettings.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     internal sealed class DefaultWebProxySettings : IWebProxy
     {
-        private static readonly DefaultWebProxySettings s_defaultWebproxy = new DefaultWebProxySettings();
```

Comment:
Does this actually work? That's pretty interesting

Reply:
C# 9.0!

I'm most excited about it for class members that are dictionary<string, object> because that gets so long.

https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-9#fit-and-finish-features